node/_tools/setup.ts

```diff
@@ -22,8 +22,30 @@ import { downloadFile } from "../../_util/download_file.ts";
  * section of the configuration file
  *
  * Usage: `deno run --allow-read --allow-net --allow-write setup.ts`
+ *
+ * You can aditionally pass a flag to indicate if cache should be used for generating
+ * the tests, or to generate the tests from scratch (-Y/-N)
  */
 
+const args = Deno.args.map((arg) => arg.toUpperCase());
```

Comment:
case insensitive flags seem a little strange to me. How about just accept `-y` and `-n`?